pkgs/development/python-modules/icontract/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+  # build
+, asttokens
+, typing-extensions
+
+  # tests
+, pytestCheckHook
+, mypy
+, pylint
+, yapf
+, tox
+, pydocstyle
+, coverage
+, docutils
+, pygments
+, dpcontracts
+, tabulate
+, py-cpuinfo
+, typeguard
+, astor
+, numpy
+, asyncstdlib
+}:
+
+buildPythonPackage rec {
+  pname = "icontract";
+  version = "2.6.1";
+  format = "setuptools";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "Parquery";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-QyuegyjVyRLQS0DjBJXpTDNeBM7LigGJ5cztVOO7e3Y=";
+  };
+
+  preCheck = ''
+    # needs to be set for tests
+    # see https://github.com/Parquery/icontract/blob/aaeb1b06780a34b05743377e4cb2458780e808d3/precommit.py#L57
+    export ICONTRACT_SLOW=1
```

Comment:
This is an assignment. 

We also do not want to run things through extra wrappers like precommit which enforce style checking which we are not interested in.

Reply:
we don't. This is why I explicitly exported `ICONTRACT_SLOW` and do not run the pre-commit script

Reply:
Can you update the comment to better reflect that? I don't think that is very clear.

Reply:
done